BrainTech/openbci
# Sensory stim

To test: connect stimulator board,
install pyftdi 0.11.3 and pyusb 1.0.0.b2
let users grab usb devices,
On systems with udev:
add file/etc/udev/rules.d/99-FTDI.rules
with line (idVendor and idProduct might be differentfor your
stimulator):
SUBSYSTEM=="usb", ATTR{idVendor}=="0403", ATTR{idProduct}=="6010", MODE="666"

Recompile azouk with multiplexer.rules from this pr.

run obci_gui --preset budzik
there should be "Haptic test" scenario, which should activate in some succession 1st and 2nd stimulation channels.

You will not see USB IO errors in obci, but if stimulator works by simply running obci/devices/haptics/HapticControl.py it SHOULD work in obci also. That file contains a class which can be used outside openbci also.
